### PR TITLE
Clean up net452 (leftover from prior prs)

### PR DESCRIPTION
**Pull request by cijothomas**
Part of https://github.com/open-telemetry/opentelemetry-dotnet/issues/2138

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

**Review by utpilla — APPROVED** [review]

File: test/OpenTelemetry.Instrumentation.Http.Tests/OpenTelemetry.Instrumentation.Http.Tests.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.1.1" Condition="'$(TargetFramework)' != 'net452'" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="1.1.1"/>
```
Comment (reyang) [inline test/OpenTelemetry.Instrumentation.Http.Tests/OpenTelemetry.Instrumentation.Http.Tests.csproj:17]: Curious, I saw that we have different versions of this package used in different places:
1. is this intentional (to have multiple versions used)?
2. do we want to extract the version to the common place?

**Comment by reyang** [comment]
There seems to be more, e.g. https://github.com/open-telemetry/opentelemetry-dotnet/blob/476e9de74e0e572af6e18c46b7f58f1bb563fbfc/test/OpenTelemetry.Exporter.Zipkin.Tests/OpenTelemetry.Exporter.Zipkin.Tests.csproj#L17

**Review by reyang — APPROVED** [review]
LGTM.

File: test/OpenTelemetry.Instrumentation.Http.Tests/OpenTelemetry.Instrumentation.Http.Tests.csproj (hunk 1 above)
Comment (cijothomas) [reply test/OpenTelemetry.Instrumentation.Http.Tests/OpenTelemetry.Instrumentation.Http.Tests.csproj]: http tests had no need of this package. it was accidently part of it. removed

**Pull request merged**